lib/util.js

```diff
@@ -202,7 +202,7 @@ util = {
             case 'event':
                 return !(entityV1.tests || entityV1.preRequestScript);
             case 'auth':
-                return _.has(entityV1, 'auth') && !(_.has(entityV1, 'currentHelper') && entityV1.helperAttributes);
+                return _.has(entityV1, 'auth') && !(_.has(entityV1, 'currentHelper') || entityV1.helperAttributes);
```

Comment:
Why this change?

Reply:
This was a bug in the legacy mode detection.

Reply:
The change above is related to this.

Reply:
Got it